cafes/views.py

```diff
@@ -42,4 +42,21 @@ def delete(self, request, cafe_id):
             return JsonResponse({'MESSAGE' : 'REVIEW_DOES_NOT_EXIST'}, status=400)
 
         review.delete()
-        return JsonResponse({'MESSAGE' : 'REVIEW_DELETED'}, status=204)
\ No newline at end of file
+        return JsonResponse({'MESSAGE' : 'REVIEW_DELETED'}, status=204)
+    
+class CommentOnReviewView(View):
+    @log_in_confirm
+    def post(self, request, review_id):
+
+        data = json.loads(request.body)
```

Comment:
```suggestion
    def post(self, request, review_id):
        data = json.loads(request.body)
```
공백 제거해 주시고 =을 기준으로 정렬해 주세요.